### DISCUSSION TITLE
[UI idea] Check if the interface also works on phones and tablets

### DISCUSSION THREAD

**Discussion by MarleenGilsing** [hedyorg/hedy#5075] [Large Projects & Plans]
Felienne just mentioned that a lot of users work on a phone or tablet. I should keep this in mind and maybe think of ways to improve the UI when using the mobile version.

**Comment by Felienne** [comment]
One very annoying issue is that on mobile, the language switcher is unusable 😭 

![IMG_5005](https://github.com/hedyorg/hedy/assets/1003685/004cef25-4498-4441-801b-9209cdc6976e)

Between 10 and 25% of users use Hedy on mobile, so this should really be fixed (although I suspect on table it looks a bit better).

**Comment by Felienne** [comment]
Maybe this specific issue (the language menu on mobile, not checking more) could be fun good new issue! I will consider splitting it out.
